modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -22,12 +22,8 @@ def perform(power_of_attorney_id, form_number, rep_id)
 
         output_path = pdf_constructor(form_number).construct(data(power_of_attorney, form_number, rep),
                                                              id: power_of_attorney.id)
-        if Flipper.enabled?(:lighthouse_claims_api_poa_use_bd)
-          doc_type = form_number == '2122' ? 'L190' : 'L075'
-          benefits_doc_api.upload(claim: power_of_attorney, pdf_path: output_path, doc_type:)
-        else
-          upload_to_vbms(power_of_attorney, output_path)
```

Comment:
Same comment as v1: do we still need `include ClaimsApi::PoaVbmsSidekiq` on line 12?